### ISSUE TITLE
Not able to create android SDK - Error - Task :react-native-device-info:compileReleaseJavaWithJavac FAILED

### ISSUE THREAD

**Issue by idrive-pankaj** [jitsi/jitsi-meet#10997]
### Description:
I have cloned the latest repository code and I am trying to create android SDK but getting this compiler error every time.
Task :react-native-device-info:compileReleaseJavaWithJavac FAILED
An exception has occurred in the compiler (1.8.0_322). Please file a bug against the Java compiler via the Java bug reporting page (http://bugreport.java.com) after checking the Bug Database (http://bugs.java.com) for duplicates. Include your program and the following diagnostic in your report. Thank you.
java.lang.AssertionError: annotationType(): unrecognized Attribute name MODULE (class com.sun.tools.javac.util.UnsharedNameTable$NameImpl)

### Steps to reproduce:

1. Clone latest master code
2. open android studio and build the project
3. hit android SDK build up command in the terminal -
` ./android/scripts/release-sdk.sh /Users/xyzUser/SDK`

### Expected behavior:

It should generate android SDK so that we can use it in the native code.

### Actual behavior:
Not able to create android SDK - even there is no single change in the current repository.

### Server information:

- Jitsi Meet version: jitsi latest code from master branch - released 24 days before
- Operating System: Mac OS Big Sur
- Android Studio - Android studio Bumblebee

### Client information:

- Browser / app version:
- Operating System:

### Additional information:

I tried to follow this similar issue - https://github.com/jitsi/jitsi-meet/issues/10975 and changed jdk settings in the android studio but the issue is still there.
<img width="964" alt="Screenshot 2022-02-22 at 2 35 35 PM" src="https://user-images.githubusercontent.com/79213346/155099040-173978c9-66a9-49da-ae72-0d2c8ec915c1.png">

**Comment by saghul** [comment]
If you build from the command line you need the JAVA_HOME environment variable pointing to a JDK 11 installation.

**Comment by idrive-pankaj** [comment]
Thanks @saghul, I have downloaded JDK 11 in my lib home folder as it was not there previously. only 1.8 was there so.
Added JAVA_HOME environment variable and it's working now.

adding images for the other users.
Thanks for the quick response @saghul 
<img width="975" alt="Screenshot 2022-02-22 at 3 04 02 PM" src="https://user-images.githubusercontent.com/79213346/155104451-4455185a-4809-446f-b58b-0f4c9da2b5a0.png">

Set JAVA_HOME variable like this.
<img width="682" alt="Screenshot 2022-02-22 at 3 07 31 PM" src="https://user-images.githubusercontent.com/79213346/155104712-2b1d9540-4944-4c83-9368-27d7efb468ab.png">
